rates/rates_test.go

```diff
@@ -175,17 +174,24 @@ func TestRateProfileCostForEvent(t *testing.T) {
 				Increments: []*utils.RateSIncrement{
 					{
 						IncrementStart:    utils.NewDecimal(0, 0),
-						Rate:              rPrf.Rates["RATE1"],
 						IntervalRateIndex: 0,
+						RateID:            "RATE1",
```

Comment:
In order to keep things close to reality here, please use UUID instead of clear IDs since it might confuse us in the future.